src/pages/ProductList/ProductList.js

```diff
@@ -1,7 +1,109 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import ProductListDetails from './ProductListDetails';
+import './ProductList.scss';
 
 const ProductList = () => {
-  return <div>ProductList</div>;
+  // const [List, setList] = useState([]);
+
+  // useEffect(() => {
+  //   fetch('http://localhost:3000/data/listData.json', {
+  //     method: 'GET',
+  //   })
+  //     .then(res => res.json())
+  //     .then(data => {
+  //       setList(data);
+  //     });
+  // }, []);
+
+  return (
+    <div>
```

Comment:
단순히 최상단을 하나의 부모 태그로 감싸기 위함이라면 React.Fragment를 활용하시면 됩니다!
```suggestion
    <React.Fragment>
```
다음처럼 쓸 수도 있습니다.
```suggestion
    <>
```